test/verify/kubelib.py

```diff
@@ -1646,20 +1646,18 @@ def testImagestreamImport(self):
         b.click("#imagestream-modify-tags")
         b.focus("#imagestream-modify-tags")
         b.key_press(['2', '.', '1', '1'])
+        b.focus("modal-dialog div.modal-footer button.btn-primary")
         b.click("modal-dialog div.modal-footer button.btn-primary")
         b.wait_not_present("modal-dialog")
 
         # new stream with only "2.11" tags should now appear
         b.wait_present('tbody[data-id="marmalade/sometags"]')
         b.wait_present("tr.imagestream-item th:contains('marmalade/sometags')")
         b.wait_in_text('tbody[data-id="marmalade/sometags"] tr', '2.11')
-        # EXFAIL: https://bugzilla.redhat.com/show_bug.cgi?id=1373332
-        # b.wait_not_in_text('tbody[data-id="marmalade/sometags"] tr', 'latest')
+        b.wait_not_in_text('tbody[data-id="marmalade/sometags"] tr', 'latest')
 
         # also check with CLI
-        output = self.openshift.execute("oc get imagestream --namespace=marmalade sometags")
-        self.assertIn("localhost:5555/juggs", output)
-        self.assertIn("2.11", output)
-        # EXFAIL: https://bugzilla.redhat.com/show_bug.cgi?id=1373332
-        # self.assertNotIn("latest", output)
+        testlib.wait(lambda: all(s in self.openshift.execute('oc get imagestream --namespace=marmalade sometags') for s in ['marmalade/sometags', '2.11']))
```

Comment:
Why does this need to check for `'marmalade/sometags'` again? the command already only queries the sometags imagestream? I. e. how does it look like when "2.11" appears, but not "marmalade/sometags"?

I don't understand at all lwhy we have to wait for this again, after the UI is already correct (in the previous paragraph). But oh well, it cannot hurt :)

Reply:
localhost:5555 is exactly the registry we're importing from. But we are importing it into marmalade yes. So maybe as soon as you import specific tags it does something different than when you import all tags from a repo. So you're right, let's just check for tags.

I'll leave the `localhost:5555/juggs` in place for the other checks, since they haven't hurt anyone :D